dao/src/main/java/org/thingsboard/server/dao/device/DeviceProfileServiceImpl.java

```diff
@@ -417,6 +423,8 @@ private void validateProtoSchemas(ProtoTransportPayloadConfiguration protoTransp
                     try {
                         validateTransportProtoSchema(protoTransportPayloadTypeConfiguration.getDeviceAttributesProtoSchema(), ATTRIBUTES_PROTO_SCHEMA);
                         validateTransportProtoSchema(protoTransportPayloadTypeConfiguration.getDeviceTelemetryProtoSchema(), TELEMETRY_PROTO_SCHEMA);
+                        validateTransportProtoSchema(protoTransportPayloadTypeConfiguration.getDeviceRpcRequestProtoSchema(), RPC_REQUEST_PROTO_SCHEMA);
```

Comment:
how do you handle backward compatibility? Is it possible to import the profile that does not have these fields? I am talking about the profile that I have exported via JSON before the upgrade?

Reply:
I have added comments below. Also, I just tried to import the profile without RPC schemas fields and it's got the default values.